Doc/library/datetime.rst

```diff
@@ -613,8 +618,8 @@ Notes:
    ``timedelta.seconds`` and ``timedelta.microseconds`` are ignored.
 
 (3)
-   This is exact, and cannot overflow. timedelta.seconds and
-   timedelta.microseconds are 0, and date2 + timedelta == date1 after.
+   This is exact, and cannot overflow. ``timedelta.seconds`` and
```

Comment:
This is a good change, but 0 on the next line doesn't need code formatting.

Reply:
I wonder if some of these `timedelta.<attr>` markups should use `:attr:` roles instead of the double backquotes used for variable names, code examples, and literals.